spec/javascripts/controllers/container_dashboard/container_dashboard_controller_spec.js

```diff
@@ -61,7 +61,33 @@ describe('containerDashboardController', function() {
                     count: 1,
                     providerType: "Kubernetes"
                 }
-            ]
+            ],
+            node_utilization:{
+                cpu:{
+                    total:8,
+                    used:3,
+                    xData:[
+                        "dates",
+                        "2015-11-26"
+                    ],
+                    yData:[
+                        "used",
+                        3
+                    ]
+                },
+                mem:{
+                    total:21,
+                    used:12,
+                    xData:[
+                        "dates",
+                        "2015-11-26"
```

Comment:
Well, if you copy the same changes here, it should merge cleanly regardless of the order. But sure, up to you :).